api/src/pcapi/core/offerers/repository.py

```diff
@@ -392,3 +395,22 @@ def find_available_reimbursement_points_for_offerer(offerer_id: int) -> list[mod
 
 def get_venue_by_id(venue_id: int) -> models.Venue:
     return models.Venue.query.get(venue_id)
+
+
+def find_offerers_validated_3_days_ago_with_no_venues() -> list[models.Offerer]:
+    three_days_ago = datetime.utcnow() - timedelta(days=3)
+    three_days_ago_min = datetime.combine(three_days_ago, time.min)
+    three_days_ago_max = datetime.combine(three_days_ago, time.max)
+
+    subquery = db.session.query(models.Venue.managingOffererId)
+
+    return (
+        db.session.query(models.Offerer)
+        .filter(models.Offerer.id.not_in(subquery))
+        .filter(
+            models.Offerer.isActive.is_(True),
+            models.Offerer.dateValidated >= three_days_ago_min,
+            models.Offerer.dateValidated <= three_days_ago_max,
```

Comment:
(détail / style / question de goût)

```suggestion
            sqla.cast(models.Offerer.dateValidated, sqla.Date) == datetime.date.today() - datetime.timedelta(days=3)
```